tests/console/btrfsmaintenance.pm

```diff
@@ -41,45 +43,80 @@ sub btrfs_service_unavailable {
     }
 }
 
+sub btrfs_await_scrub {
+    if (is_sle("<15-SP4")) {
+        script_retry('btrfs scrub status / | grep -e "scrub started .* and finished after"', retry => 20, delay => 30);
```

Comment:
One minor comment,  It seems retry many times with short `delay` time for these functions,
Does it make sense to reduce the `retry` value and increase the "delay` time?

Reply:
I think this is a duplicate, so I'm marking it as resolved.